raiden/api/python.py

```diff
@@ -193,25 +193,25 @@ def get_transfer_from_task(
 
 
 def transfer_tasks_view(
-    transfer_tasks: Dict[SecretHash, TransferTask],
+    payment_mapping: Dict[Address, PaymentMappingState],
```

Comment:
`address_to_payment_mapping`, or `address_payment_mappings` seem (to me) better choices for this field name

Reply:
Yeah, i will do that refactor after the acceptance criteria of this pr is met.

Reply:
We changed this to payment_states_by_address since it's a dict of PaymentMappingState by Address key